### PR TITLE
Add Java balance example

### PR DESCRIPTION
**Pull request by samuel-rufi**
Ref: https://github.com/iotaledger/iota-sdk/issues/50

### CODE REVIEW

**Review by Thoralf-M — DISMISSED** [review]

File: client/bindings/java/examples/src/node_api_core/GetAddressBalance.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                }
+            }
+
+            total = o.toJson().get("amount").getAsInt();
+        }
+
+        System.out.println("total balance: " + total);
+        System.out.println("native tokens: " + nativeTokens);
+
+        System.out.println(addresses[0]);
```
Comment (samuel-rufi) [inline client/bindings/java/examples/src/node_api_core/GetAddressBalance.java]:
```suggestion
```

**Review by Thoralf-M — DISMISSED** [review]

File: documentation/docs/libraries/java/how_to/_06_get_address_balances/_code_sample.mdx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import CodeBlock from '@theme/CodeBlock';
+import GetOutput
```
Comment (thibault-martinez) [inline documentation/docs/libraries/java/how_to/_06_get_address_balances/_code_sample.mdx]: GetOutput ?

Comment (samuel-rufi) [inline documentation/docs/libraries/java/how_to/_06_get_address_balances/_code_sample.mdx]:
```suggestion
import GetAddressBalance
```

File: documentation/docs/libraries/java/how_to/_06_get_address_balances/_code_sample.mdx [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+    from '!!raw-loader!../../../../../../client/bindings/java/examples/src/node_api_core/GetAddressBalance.java';
+
+<CodeBlock className="language-java">
+    {GetOutput}
```
Comment (samuel-rufi) [inline documentation/docs/libraries/java/how_to/_06_get_address_balances/_code_sample.mdx]:
```suggestion
    {GetAddressBalance}
```

File: client/bindings/java/examples/src/node_api_core/GetAddressBalance.java [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        Map<String, Integer> nativeTokens = new HashMap();
+        for (Map.Entry<Output, OutputMetadata> entry : outputs) {
+            Output o = entry.getKey();
+
+            if (o.toJson().has("nativeTokens")) {
+                for(JsonElement elem: o.toJson().get("nativeTokens").getAsJsonArray()) {
+                    NativeToken nativeToken = new NativeToken(elem.getAsJsonObject());
+                    String tokenId = nativeToken.toJson().get("id").getAsString();
+                    String amount = nativeToken.toJson().get("amount").getAsString().replace("0x", "");
+                    nativeTokens.put(tokenId, Integer.parseInt(amount, 16));
```
Comment (thibault-martinez) [inline client/bindings/java/examples/src/node_api_core/GetAddressBalance.java]: Does this also do an addition if it already exists ?

File: client/bindings/java/examples/src/node_api_core/GetAddressBalance.java [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+                }
+            }
+
+            total = o.toJson().get("amount").getAsInt();
```
Comment (thibault-martinez) [inline client/bindings/java/examples/src/node_api_core/GetAddressBalance.java]: No addition ?

Comment (samuel-rufi) [reply client/bindings/java/examples/src/node_api_core/GetAddressBalance.java]: good catch

**Review by thibault-martinez — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**